config/fortify-client.properties

```diff
@@ -0,0 +1 @@
+fortify.client.releaseId=115830
```

Comment:
The default fortify scan exclusions (see https://github.com/hmcts/ccd-data-store-api/blob/master/config/fortify-client.properties) include the test, integrationTest and functionalTest directories.  The SDT code uses differently named directories for tests, specifically unit-test, integ-test and end-to-end-test.  Do we need to add custom exclusions for these directories here?  If so, do we need to add them for each sub-project or can we use some sort of wildcard path matching (e.g. **/unit-test).